### PR TITLE
Moar UI goodness

### LINKED ISSUE #247: The round progress indicator is hidden by the new highlighted continue button
We could try to use another color, or increase the indicator size.

### PR DESCRIPTION
**Pull request by mvaivre**
Tentatively fixes #247 #258 #262 #255 #259

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -105,6 +105,12 @@ PODS:
     - ExpoModulesCore
   - EXScreenCapture (5.3.0):
     - ExpoModulesCore
+  - EXSensors (12.3.0):
+    - ExpoModulesCore
+  - EXSplashScreen (0.20.5):
```
Comment (nop33) [inline ios/Podfile.lock:110]: This splash screen keeps making a comeback 😆

File: src/components/QRCodeScannerModal.tsx [hunk 2]
```diff
@@ -91,9 +91,9 @@ const QRCodeScannerModal = ({ onClose, onQRCodeScan, qrCodeMode = 'simple', text
     <ScreenSectionCentered fill>
       {text && (
         <TextContainer>
-          <AppText size={16} semiBold color="white">
+          <AppTextCentered size={16} semiBold color="white" style={{ textAlign: 'center' }}>
```
Comment (nop33) [inline src/components/QRCodeScannerModal.tsx:94]: If it gets centered by the style prop, why is the component name change needed?

File: src/screens/DashboardScreen.tsx [hunk 3]
```diff
@@ -69,59 +71,88 @@ const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
...
+              short
               flex
             />
-          </ButtonsRow>
-        </ButtonsRowContainer>
+          </ButtonsRowContainer>
+        )}
       </BalanceAndButtons>
       <AddressesTokensList />
+      {totalBalance === BigInt(0) && (
```
Comment (nop33) [inline src/screens/DashboardScreen.tsx:115]: `totalBalance === 0` works as well (we have both variants in this component, let's choose one for consistency)

File: src/screens/new-wallet/DecryptScannedMnemonicScreen.tsx [hunk 4]
```diff
@@ -100,27 +110,37 @@ const DecryptScannedMnemonicScreen = ({ navigation }: DecryptScannedMnemonicScre
     setError('')
   }
 
+  const handleBlur = () => {
+    // And this as well is to aoid weird iOS issue where input is focused then blurred right away
+    // WTF.
```
Comment (nop33) [inline src/screens/new-wallet/DecryptScannedMnemonicScreen.tsx:115]: 😆

**Review by nop33 — APPROVED** [review]
Oops, I added my comments as comments and not review feedback. Anway, all looks good apart from those minor things, approving.

File: ios/Podfile.lock (hunk 1 above)
Comment (mvaivre) [reply ios/Podfile.lock:110]: Yeah I decided not to remove it, as it seems that it's a default :/

**Pull request merged**